types/index.d.ts

```diff
@@ -344,6 +345,9 @@ declare module 'cloudinary' {
         sign_url?: boolean;
         use_root_path?: boolean;
         auth_token?: object;
+        account_id?: string;
+        provisioning_api_key: string;
+        provisioning_api_secret: string;
```

Comment:
Hi @patrick-tolosa! Why these two are required? My type checking start complaining about them.

Reply:
Hi,

Thanks for bringing this up, I've opened #451  to address it.

Reply:
thank you!